### ISSUE TITLE
midterm Peer Review

### ISSUE THREAD

**Issue by xinciliu** [jiangckevin/ORIE-4741-Learning-with-Big-Messy-Data-Project#6]
Summary:
The goal of this project is to build up a classification model to classify whether people are having health issues and are not seeking treatment. The researchers want to use features like age, gender, self_employment, family_history, no_employee, remote_work and other indicators to build up classification models. 
What I like for their project:
1. They did one-hot encoding for most of their features. 
2. They used random forest to get the effectiveness of different features. This is important because the number of features is relatively large. 
3. They mentioned their evaluation method and the loss function they are going to use.
What I’m concerned about their project:
1. They only mentioned the random forest method, they should consider more possible models in future work if the RF model have some disadvantages.
2. They mentioned that they will change the number of features if model is under-fit or overfit. But only changing the number of features may have some bad effects. (Since some features are useless, including the useless features will be harmful). 
3. They should consider the unbalance data, which means that people who is having health issues and are not seeking treatment maybe far less than people who don’t have health issue or is already seeking for treatment.
